Cambios en correo de devolución

CarlosVzzConsiss opened the pull request:
Se agregan correos de Mauricio y Yazmin en el correo de devolución, además se agregan validaciones para campos recibidos.

CarlosVzzConsiss committed:
Cambios en correo de devolución

Se agregan correos de Mauricio y Yazmin en el correo de devolución, además se agregan validaciones para campos recibidos.

RubenConsiss reviewed (changes requested):
Favor remover el código hardocode del to: y bcc: en el envío de correo.
Consideraciones: 
-Los pull request son para fusionar código productivo.
-Todo el hardocode es para pruebas locales.

CarlosVzzConsiss committed:
Cambios fecha de Inicio

Se agregaron parámetros para mostrar en el cuerpo del correo de recepción la fecha y hora de inicio del proceso.

CarlosVzzConsiss committed:
Remover hardcode

Se remueven los correos hardcode de Ruben y Carlos.

CarlosVzzConsiss reviewed (commented):
Listo, ya se hicieron los ajustes.
